commands/remove.go

```diff
@@ -40,7 +40,12 @@ func runDelete(cmd *cobra.Command, args []string) error {
 	var services stack.Services
 	var gatewayAddress string
 	var yamlGateway string
-	if len(yamlFile) > 0 {
+
+	argsSelection := false
+
+	if len(args) == 1 {
```

Comment:
What if you have both args and YAML?

Reply:
It prefers args and does not perform the operation on the `stack.yml`. I think that is more expected by the user.